### PR TITLE
UI of AniFetch

### PR DESCRIPTION
**Pull request by abhinavcodedev**
# Pull Request

## Description

Please include a summary of the changes you have made. Highlight the key aspects of your project.

## Checklist

- [ ] My project includes a `README.md` file with a brief overview of the project, how to set it up, and any relevant information.
- [ ] I have added a screenshot or screen recording showing the functionality of the app.
- [ ] I have tested the project and confirmed it works as expected.

## Screenshots

If applicable, please add a screenshot or a screen recording to demonstrate the functionality of your app.

## Project README

Please ensure that your project's `README.md` is detailed and includes:

- A brief introduction to the project.
- Installation steps.
- Key features.
- Any additional instructions or information.

## Additional Notes

Is there anything else the maintainers should know about your contribution?

---

Thank you for your contribution!

### CODE REVIEW

File: AniFetch/Screenshot_20240821_143126.png
Comment (anandwana001) [inline AniFetch/Screenshot_20240821_143126.png:1]: can we add screenshot to pr descritpion as well?

File: AniFetch/app/src/main/java/com/example/anifetch/APIServices.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.example.anifetch
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+interface APIServices {
+    @GET("anime")
```
Comment (anandwana001) [inline AniFetch/app/src/main/java/com/example/anifetch/APIServices.kt:10]: not a day 1 task

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Currently, we are reviewing and accpeting ony day 1 task

File: AniFetch/app/src/main/java/com/example/anifetch/APIServices.kt (hunk 1 above)
Comment (abhinavcodedev) [reply AniFetch/app/src/main/java/com/example/anifetch/APIServices.kt:10]: So can I remove the API service from my code and than use local storage. First I check with local storage then I use API

**Comment by anandwana001** [comment]
@abhinavcodedev update PR description with all the details

File: AniFetch/app/src/main/java/com/example/anifetch/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            AnifecthTheme {
+                //navigate to animedetailScreen
+                val navController = rememberNavController()
+                NavHost(navController =navController, startDestination = "main") {
+                    composable("main"){ MainScreen(navController)}
+                    composable("animeDetail/{animeId}"){backStackEntry->
+                        val animeId = backStackEntry.arguments?.getString("animeId")?.toInt() ?: return@composable
+                        AnimeDetailScreen(animeId,navController)
+                    }
+                }
```
Comment (anandwana001) [inline AniFetch/app/src/main/java/com/example/anifetch/MainActivity.kt]: should be in a separate file

File: AniFetch/app/src/main/java/com/example/anifetch/MainActivity.kt [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+            }
+        }
+    }
+}
+
+@Composable
+fun MainScreen(navController: NavHostController) {
```
Comment (anandwana001) [inline AniFetch/app/src/main/java/com/example/anifetch/MainActivity.kt]: separate file

File: AniFetch/app/src/main/java/com/example/anifetch/MainActivity.kt [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+        //SearchIcon
+        /*Icon(imageVector = Icons.Default.Search,
+            contentDescription = "Search",
+            modifier = Modifier
+                .size(30.dp)
+                .align(Alignment.CenterStart)
+                .padding(end = 16.dp),
+            tint = MaterialTheme.colorScheme.onSurface
+        )*/
+            //Basic Text Field for search input
```
Comment (anandwana001) [inline AniFetch/app/src/main/java/com/example/anifetch/MainActivity.kt]: why comment code in PR?

**Review by anandwana001 — CHANGES_REQUESTED** [review]
For Now, I am reviewing only UI part as part of Day 1 task.

Please clean the UI Code.

**Comment by abhinavcodedev** [comment]
@anandwana001  I've addressed the requested changes. Please review when you get a chance.

**Review by anandwana001 — APPROVED** [review]

**Pull request merged**